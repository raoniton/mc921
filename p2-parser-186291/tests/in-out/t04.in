class Main {
    public static void main(String[] args) {
        int a = 100 + 50;
        int b = (13 + a) * 10;
    }
}