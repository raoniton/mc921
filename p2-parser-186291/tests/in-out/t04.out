Program:
    ClassDecl: ID(name=Main) @ 1:1
        MainMethodDecl: @ 2:5
            ID: args @ 2:38
            Compound: @ 2:44
                VarDecl: ID(name=a) @ 3:13
                    Type: int @ 3:9
                    BinaryOp: + @ 3:17
                        Constant: int, 100 @ 3:17
                        Constant: int, 50 @ 3:23
                VarDecl: ID(name=b) @ 4:13
                    Type: int @ 4:9
                    BinaryOp: * @ 4:17
                        BinaryOp: + @ 4:18
                            Constant: int, 13 @ 4:18
                            ID: a @ 4:23
                        Constant: int, 10 @ 4:28
